engine/src/main/java/org/terasology/persistence/internal/SaveTransaction.java

```diff
@@ -138,6 +144,8 @@ public void run() {
             mergeChanges();
             result = SaveTransactionResult.createSuccessResult();
             logger.info("Save game finished");
+            GameManifest latestManifest = getLatestGameManifest();
+            storageManager.setSavePath(PathManager.getInstance().getSavePath(latestManifest.getTitle()));
```

Comment:
Not sure if:
* It is a good idea to have 2 classes use each (SaveTransaction and storageManger) other. Maybe it would be cleaner if for example the storageManger would update it's save path itself.
* If changing the save path is design wise the correct approach. Maybe StorageManager could have a `saveAs` method that takes a path and does a regular save and then copy the save to that location.
* That changing the save path will create a proper copy as far as I can remember the save system saves only the chunks that have changed.